### PR TITLE
Add Gitpod version and score info metrics

### LINKED ISSUE #4182: New prometheus info metric that tells the current score of a cluster.
After the introduction of pluggable workspace clusters, we noticed that our dashboard `Gitpod / Overview` doesn't scale as well as Gitpod 😛. See https://github.com/gitpod-com/observability/issues/100 for details.

We want to do a revamp of that dashboard and one of the things we would like to have there is the current score of each cluster.

@csweichel @geropl do you know which component would be the one that holds such information?

### PR DESCRIPTION
**Pull request by ArthurSens**
Signed-off-by: ArthurSens <arthursens2005@gmail.com>

## Description

When ready, this PR will add two metrics that will be shown in `Gitpod / Overview` dashboard.
1. Gitpod's version, exposed by server.

## How to test

Hit the metrics endpoint of server  and look for the metric:
1. `gitpod_version_info`

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@ArthurSens: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

Associated issue: *#4182*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

Associated issue: *#4182*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**
- **[components/ws-manager-bridge/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager-bridge/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/ws-manager-bridge/src/main.ts
Comment (ArthurSens) [inline components/ws-manager-bridge/src/main.ts:34]: Logic still needs to be implemented here

File: components/ws-manager-bridge/src/cluster-service-server.ts
Comment (ArthurSens) [inline components/ws-manager-bridge/src/cluster-service-server.ts:391]: I'mm getting an error here:
```
(node:30) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'findFiltered' of undefined\n    at ClusterServiceServer.<anonymous> (/app/node_modules/@gitpod/ws-manager-bridge/dist/src/cluster-service-server.js:528:61)\n    at step (/app/node_modules/@gitpod/ws-manager-bridge/dist/src/cluster-service-server.js:60:23)\n    at Object.next (/app/node_modules/@gitpod/ws-manager-bridge/dist/src/cluster-service-server.js:41:53)\n    at /app/node_modules/@gitpod/ws-manager-bridge/dist/src/cluster-service-server.js:35:71\n    at new Promise (<anonymous>)\n    at __awaiter (/app/node_modules/@gitpod/ws-manager-bridge/dist/src/cluster-service-server.js:31:12)\n    at ClusterServiceServer.getScore (/app/node_modules/@gitpod/ws-manager-bridge/dist/src/cluster-service-server.js:513:16)\n    at /app/node_modules/@gitpod/ws-manager-bridge/dist/src/main.js:84:52\n    at new Promise (<anonymous>)\n    at /app/node_modules/@gitpod/ws-manager-bridge/dist/src/main.js:83:21\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at next (/app/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/app/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)\n    at /app/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/app/node_modules/express/lib/router/index.js:335:12)
```

Which makes sense, since I'm not declaring this `clusterDB` object anywhere. Could anyone provide any tips on how to make this work?

Comment (corneliusludmann) [reply components/ws-manager-bridge/src/cluster-service-server.ts:391]: I think you need an inject (for the dependency injection) as here:

https://github.com/gitpod-io/gitpod/blob/66aa4bd6c2f9299df8429ebc4f4179e75de55aa7/components/ws-manager-bridge/src/cluster-service-server.ts#L49-L50

File: components/ws-manager-bridge/src/cluster-service-server.ts
Comment (ArthurSens) [inline components/ws-manager-bridge/src/cluster-service-server.ts:398]: No errors, but I'm seeing this now:
image


I'm assuming that happens because I haven't passed a proper filter to `clusterDB.findFiltered` function. 

How can I use the filter to find only the cluster where this ws-manager-bridge runs on?

Comment (csweichel) [reply components/ws-manager-bridge/src/cluster-service-server.ts:398]: This code should be in `BridgeController` - there you'll also find examples how to filter the workspace cluster

**Comment by ArthurSens** [comment]
Okey nice. Looks like everything is working as expected:

image
image
image

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

Associated issue: *#4182*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ArthurSens** [comment]
After https://github.com/gitpod-io/gitpod/pull/5726, I've updated this PR to only expose the version metric.

PTAL @csweichel

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 492d5d16af6dc4d2af000f54ca011b61edbdbef1</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5557#issuecomment-921804072" title="LGTM">csweichel</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by JanKoehnlein — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5557#pullrequestreview-757475858" title="Approved">JanKoehnlein</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5557#issuecomment-921806244" title="Approved">JanKoehnlein</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5557#issuecomment-921806244" title="Approved">JanKoehnlein</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
